geoid/__init__.py

```diff
@@ -122,7 +124,7 @@
     'sdsec': 5,  # State-School District (Secondary)
     'sduni': 5,  # State-School District (Unified)
     'sldl': 3,  # State Legislative District Lower
-    'sldu': "3",  # State Legislative District Upper
```

Comment:
Ok. I was confused by the name of the dict. Looks like it contains lengths mapping. Also the comment "# Lengths in number of decimal digits" seems invalid.